autoload/classpath.vim

```diff
@@ -0,0 +1,37 @@
+let s:entry = "\t<classpathentry kind=\"<kind>\" path=\"<path>\"/>"
+let s:entry_sourcepath = "\t<classpathentry kind=\"<kind>\" path=\"<path>\" sourcepath=\"<src>\"/>"
+
+" Creates a .classpath file and fills it with dependency entries
+function! classpath#generateClasspath() abort
+  let l:classes = extend(gradle#classPaths(), android#classPaths())
+  let classpath = ['<?xml version="1.0" encoding="UTF-8"?>', '<classpath>']
+  for jar in l:classes
+    call add(classpath, classpath#NewClasspathEntry('lib', jar))
+  endfor
+  call add(classpath, classpath#NewClasspathEntry('src', 'src/main/java'))
+  call add(classpath, '</classpath>')
+  call writefile(classpath, 'app/.classpath')
+endfunction
+
+" Adds a new entry to the current .classpath file.
+function! classpath#NewClasspathEntry(kind, arg, ...)
```

Comment:
This function does not need to be called outside the script. Rename it to:  `s:newClassEntry()`.